### ISSUE TITLE
CAN motor controllers don't move and report timeout/stale CAN messages

### ISSUE THREAD

**Issue by calcmogul** [wpilibsuite/sysid#402]
We've been getting widespread reports of CAN motor controllers not moving after either SysId binary (C++ with an RT thread) is deployed. It's gotten so bad that people just aren't using the tool, and most teams use CAN motor controllers.

We've confirmed they're using the new roboRIO image, and the console prints messages about CAN message receive timeouts and the messages being stale. The robots under test generally only had like 4 motors installed, so CAN bus overload is unlikely. We've gotten a lot more reports with Falcons than with NEOs, but that could be due to relative marketshare or the proportion of each group of teams that try the tool and mention to us that they had issues.

3512 has gotten stale messages in non-SysId robot code too (C++ with RT thread) with Spark Maxes even after increasing all the non-essential CAN bus status frame periods to 500ms. CAN bus utilization was also low (I wasn't told exactly how much). I don't know if it was ever addressed.

At this point, I think the most likely cause is the vendordeps not playing nicely together since the SysId binary links them all in.

WPILib has no physical hardware, so we have no way to troubleshoot this. One way someone with hardware could try to narrow down the issue is remove vendordeps from sysid-projects and rebuild/redeploy until the problem goes away.

**Comment by calcmogul** [comment]
3512 has 9 NEOs, and they also make TimedRobot run at 5ms. This wasn't a problem in 2020/2021, but seems to be an issue for 2022.

**Comment by modelmat** [comment]
Have confirmation from team 5032 [here](https://discord.com/channels/176186766946992128/528555967827148801/956699320210579547) that a [version of the code](https://cdn.discordapp.com/attachments/528555967827148801/956697266872279050/sysid-only-ctre.exe) with only the CTRE vendordep works.
Going to push the source for that .exe in just a second.

**Comment by trevnels** [comment]
6502 may have seen similar behavior during this past season on Spark MAXes. If I remember correctly, we saw CAN timeouts after deploying sysid, but motors _would_ move when enabled - just in an erratic fashion (as if they were quickly starting and stopping.) Fully power cycling the robot after deploying sysid usually resolved it. 

Back then, my hunch was that there was something going wrong during the transition between previously deployed code and sysid which was fixed by a reboot, though I'm not familiar enough with sysid's internals to know if this theory makes sense.
